databrowser/src/domain/datasets/ui/toolBox/ToolBoxSection.vue

```diff
@@ -0,0 +1,19 @@
+<!--
+SPDX-FileCopyrightText: NOI Techpark <digital@noi.bz.it>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<template>
+  <slot />
+</template>
+
+<script setup lang="ts">
+import type { Component } from 'vue';
+
+defineProps<{
+  sectionKey: string; // meglio evitare `key` perché è una parola speciale di Vue
+  title: string;
+  iconComponent?: Component;
+  info?: string;
```

Comment:
This prop is not used by [ToolBox.vue](https://github.com/dallefeste/opendatahub-databrowser/blob/61b8365b43fbf85592c8a0dc73165b987e4eb1e4/databrowser/src/domain/datasets/ui/toolBox/ToolBox.vue), so I would remove it